rustls-libssl/src/lib.rs

```diff
@@ -615,6 +616,13 @@ impl SslContext {
         self.verify_x509_store.pointer()
     }
 
+    fn set_x509_store(&mut self, store: *mut X509_STORE) {
+        self.verify_x509_store = match store.is_null() {
+            true => OwnedX509Store::default(),
+            false => OwnedX509Store::new(store),
```

Comment:
Just want to call out that the semantics of `SSL_CTX_set_cert_store` are to take store as a donated existing ref.  There's a separate `SSL_CTX_set1_cert_store` (not impl'd) that will upref store.

In both cases the existing store (if any) is to be freed, which should happen automatically from the `Dropl` impl.